packages/flutter/lib/src/services/_background_isolate_binary_messenger_io.dart

```diff
@@ -0,0 +1,97 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async' show Completer;
+import 'dart:isolate' show ReceivePort;
```

Comment:
This is the important line, now `isolate` is only brought in through a conditional import.